app/templates/MyJobPosts.html

```diff
@@ -2,112 +2,105 @@
-                    <p><strong> Applicants:</strong> </p>
-                    {% for applicant in applicant_info %}
-                        {% if applicant[0].post_id == post.id %}
-                            <div style="display: inline-flex;">
-                                <p><b>{{applicant[2].first_name}} {{applicant[2].last_name}}</b></p> 
-                                <p><button class = 'btn-sm' onclick="applicationExpand('applicant_box_{{ loop.index }}')" id = "applicant_btn_{{loop.index}}">+</button></p>
-                            </div>
-                            <div id = "applicant_box_{{ loop.index }}" style = "display: none;">
-                                <p>Title: {{ applicant[1].title_apl }}</p>
-                                <p>Contact at: {{ applicant[2].email }}</p>
-                                <p>Earliest Start Date: {{ applicant[1].earliest_start_date }}</p>
-                                <p>Personal Bio: {{ applicant[1].personal_bio }}</p>
-                                <p>Health Status: {{ applicant[1].health }}</p>
-                                <p>Cover Letter: {{ applicant[0].cover_letter }}</p>
-                            </div>
-                            <br>
-                        {% endif %}
-                    {% endfor %}
-
-                    <!-- applying for job button-->
-                        <!-- application form-->
-                </div>
+
+        <!-- create a feedbox object -->
+        <div class="feedBox feedPageBox feedBoxSwell">
+
+            <!-- words for post title -->
+            <h1> {{post.title}}
+
+                <!-- label for post title and expandable content-->
+                <span>
+
+                    <!-- stylying button with 3 different selectors -->
+                    <button onclick="toggleExpand('expand_btn_{{ loop.index }}')" class = "btn btn-sm btn-outline-info feed_expand_btn" id = "expand_btn_{{ loop.index }}">
+
+                        <!-- image of plus icon as defualt "+" rendering inappropriate-->
+                        <img src="../static/plusIcon.png" alt="" style="width: 20px; height: 22px;">
+                    </button>
+                </span>
+            </h1>
+
+            <!-- Metadata -->
+            <p><strong> {{ post.date_posted }}</strong> </p>
+            <!-- Expandable content section -->
+            <div class="expandable_content" id= "expandable_content_{{loop.index}}" style="display: none;">
+                <p><strong> Applicants:</strong> </p>
+                {% for applicant in applicant_info %}
+                    {% if applicant[0].post_id == post.id %}
+                        <div style="display: inline-flex;">
+                            <p><b>{{applicant[2].first_name}} {{applicant[2].last_name}}</b></p> 
+                            <p><button class = 'btn-sm' onclick="applicationExpand('applicant_box_{{ loop.index }}')" id = "applicant_btn_{{loop.index}}">+</button></p>
```

Comment:
would be nice to see more styling of this button to ensure consistency with other buttons in feed page that have a similar purpose to ensure document consistency.